src/test/java/edu/cornell/kfs/sys/web/mock/MockRemoteServerExtension.java

```diff
@@ -0,0 +1,75 @@
+package edu.cornell.kfs.sys.web.mock;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpHost;
+import org.apache.http.localserver.LocalServerTestBase;
+
+import edu.cornell.kfs.sys.web.mock.MockServiceEndpointBase;
+
+// TODO: Convert this into a true JUnit 5 test extension class.
```

Comment:
should we create a follow up user story and remove this comment?

Reply:
I'll remove this comment, and update the title of sub-task KFSPTS-23928 to have an appropriate follow-up user story created for this.